### PR TITLE
Release 2.3.0 documentation

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Signed-off-by: Orlin Vasilev <ovasilev@vmware.com>

### CODE REVIEW

**Comment by a-mccarthy** [comment]
@OrlinVasilev the config file changes look good to me. Something not documented in the release instructions is the need to update the docs/_index.md file to bump the version to 2.3 in the master branch. 

We also need to have the release-2.3.0 branch created before merging the config.toml file changes, or else we'd get build errors.

**Comment by OrlinVasilev** [comment]
@a-mccarthy - bumped the version in _index.md , and added that instruction to the README.md.
branch release-2.3.0 is created as well!

File: README.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ helmversion = "1.3"
 branchname = "release-X.Y.Z"
 ```
 
+As last step need to bump the [_index.md](https://github.com/goharbor/website/blob/master/docs/_index.md) file with the new version and fix the links in the file
```
Comment (jonasrosland) [inline README.md:39]: Could be "As the last step, update the ... file with the new version number and correct links."

Comment (OrlinVasilev) [reply README.md]: done!

**Review by a-mccarthy — APPROVED** [review]
lgtm

**Pull request merged**